compiler_setup/ompss.sh

```diff
@@ -0,0 +1,36 @@
+# Build settings for gfortran compiler
+F90=mpifort
+CC=mpicc
+CXX=mpic++
+
+# Common optimization flags for CFLAGS and F90FLAGS
+OPTFLAGS=" -O3 -mtune=native -finline-limit=50000 -fopt-info-all=gnu_opt_report.txt"
+OPTFLAGS+=" -g"
+
+CFLAGS=$OPTFLAGS
+F90FLAGS="-Wall -Wsurprising -Wuninitialized"
+#F90FLAGS += -O0
+#F90FLAGS += -fcheck=all -fbacktrace -ffpe-trap=invalid -g
+F90FLAGS+=" -faggressive-function-elimination"
+F90FLAGS+=$OPTFLAGS
+# f2py does not break long lines so tell gfortran not to
```

Comment:
You can delete these 3 lines as we do now break lines where appropriate.

Reply:
Done

Reply:
They're still there...?

Reply:
Yup I removed the wrong line, will be fixed in next commit.